json-rpc/src/methods.rs

```diff
@@ -124,7 +124,7 @@ type RpcHandler =
 pub(crate) type RpcRegistry = HashMap<String, RpcHandler>;
 
 pub(crate) struct JsonRpcRequest {
-    pub trace_id: u64,
+    pub trace_id: String,
```

Comment:
any point in changing this? it allows weird stuff to be inserted here

can we not just convert the trace id to a string in the logger?

Reply:
I figured the hex string would be easier to read and more compact / consistent.  When stringifying a number with the highest bit set, it changes the number from a base 10 to a big int e.g. 1e10 or whatever.